valusets/ValueSet-England-PractitionerRoleCode.xml

```diff
@@ -0,0 +1,31 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-PractitionerRoleCode"></id>
+    <url value="https://fhir.nhs.uk/ValueSet/England-PractitionerRoleCode"></url>
+    <version value="1.0.0"></version>
+    <name value="EnglandPractitionerRoleCode"></name>
+    <title value="England PractitionerRole Code"></title>
+    <status value="draft"></status>
+    <date value="2023-06-01"></date>
+    <publisher value="NHS England"></publisher>
+    <contact>
+        <name value="NHS England"></name>
+        <telecom>
+            <system value="email"></system>
+            <value value="interoperabilityteam@nhs.net"></value>
+            <use value="work"></use>
+			<rank value="1"></rank>
+        </telecom>
+    </contact>
+    <description value="PractitionerRole.code definition"></description>
+    <compose>
+        <include>
+            <system value="https://fhir.nhs.uk/CodeSystem/England-SDSJobRoleCode"></system>
+        </include>
+        <include>
+            <system value="https://fhir.hl7.org.uk/CodeSystem/UKCore-SDSJobRoleName"></system>
```

Comment:
This is a retired CodeSystem, should we be using it?

Reply:
I did notice and wasn't sure myself too. I included it because there is a fixedUri value in PractitionerRole. So if I removed the CodeSystem then I suppose I would need to remove below from PractitionerRole: `<element id="PractitionerRole.code.coding:SDSJobRoleName.system">
      <path value="PractitionerRole.code.coding.system" />
      <fixedUri value="https://fhir.hl7.org.uk/CodeSystem/UKCore-SDSJobRoleName" />
    </element>`

What do you think?

Reply:
See above comment to progress.

Reply:
@APageNHS @KazeemHamzat I have changed the status of ValueSet SDSJobRoleCode from Retired to Draft. This was agreed for the time being in the conversation in Slack.